verilog/tools/syntax/export_json_examples/verible_verilog_syntax.py

```diff
@@ -370,11 +370,58 @@ class VeribleVerilogSyntax:
   This class provides methods for running ``verible-verilog-syntax`` and
   transforming its output into Python data structures.
 
+  Attributes:
+    executable (str): path to ``verible-verilog-syntax`` binary.
+    version (str): ``verible-verilog-syntax`` version
+
   Args:
     executable: path to ``verible-verilog-syntax`` binary.
   """
 
-  def __init__(self, executable: str = "verible-verilog-syntax"):
+  _VERSION_RE = re.compile(r"^v([0-9]+)\.([0-9]+)-([0-9]+)-g[0-9a-fA-F]+$",
+      re.MULTILINE)
+  _EXPORT_JSON_RE = re.compile(r"(^|\s)--export_json\b")
+  _MIN_VERSION = "v0.0-925-gc1a388a"
+
+  def __init__(self, executable: Union[str, List[str]] = "verible-verilog-syntax"):
+    if isinstance(executable, str):
+      executable = [executable]
+
+    proc = subprocess.run([*executable, "--version"], capture_output=True,
```

Comment:
when I run this in an environment with a 3.6.13 Python, I get the following error
```
TypeError: __init__() got an unexpected keyword argument 'capture_output'
```

Is this a feature that is only available in newer Python versions ?

Reply:
Yes, it has been introduced in 3.7. I changed it to `stdout`/`stderr` - should work with 3.6.